tools/lsp/preview/native.rs

```diff
@@ -99,12 +99,6 @@ pub fn quit_ui_event_loop() {
     close_ui();
 
     let _ = i_slint_core::api::quit_event_loop();
-
-    // Make sure then sender channel gets dropped
-    if let Some(sender) = SERVER_NOTIFIER.get() {
-        let mut sender = sender.lock().unwrap();
-        *sender = None;
-    };
```

Comment:
If you don't do that, there is a risk that some of the threads within the lsp-server don't quit and that because of that, the slint-lsp process stays alive forever.  
Are you sure it's OK to remove?